### PR TITLE
Add support for optional logarithmic axes

### PR DESCRIPTION
**Pull request by kristofferlund**
##### Checklist

- [x] PR description included and it is linked to an issue
- [x] `make test` and `make lint` passes
- [x] relevant documentation is changed or added

Notes: Demonstrated in line chart example

### CODE REVIEW

File: visualizations/fan_chart/webviz_fan_chart/__init__.py [hunk 1]
```diff
@@ -182,12 +182,16 @@ class FanChart(FilteredPlotly):
         Expects `index` parameter to be used as 'x' value, a `name` parameter
         to correspond with a name in the data dataframe, a `value` and `value`
         that will determine the size of the marker (in height)
-    :param kwargs: optional `xaxis` and `yaxis` paramameter. Will create a
-        label for the given axis. Defaults to `None`.
+    :param xaxis: Will create a label for the x-axis. Defaults to `None`.
+    :param yaxis: Will create a label for the y-axis. Defaults to `None`.
+    :param logx: boolean value to toggle x-axis logarithmic scale
+    :param logy: boolean value to toggle y-axis logarithmic scale
```
Comment (anders-kiaer) [inline visualizations/fan_chart/webviz_fan_chart/__init__.py]: Add information regarding default value? E.g.

`:param logx: If to use logarithmic x-axis or not. Defaults to `False`.`

Comment (kristofferlund) [reply visualizations/fan_chart/webviz_fan_chart/__init__.py]: Done 👍

File: visualizations/bar_chart/webviz_bar_chart/__init__.py [hunk 2]
```diff
@@ -13,19 +13,22 @@ class BarChart(FilteredPlotly):
         ``'overlay'``. Defines how multiple bars per index-value are combined.
         See `plotly.js layout-barmode <https://plot.ly/javascript/reference/
         #layout-barmode>`_.
-    :param kwargs: optional `xaxis` and `yaxis` paramameter. Will create a
-        label for the given axis. Defaults to `None`.
+    :param xaxis: Will create a label for the x-axis. Defaults to `None`.
+    :param yaxis: Will create a label for the y-axis. Defaults to `None`.
+    :param logy: boolean value to toggle y-axis logarithmic scale.
+        Defaults to False
```
Comment (anders-kiaer) [inline visualizations/bar_chart/webviz_bar_chart/__init__.py]: → Change from False to `False` (i.e. with back ticks) to get same style as with `None` above?

**Comment by kristofferlund** [comment]
I added a handler for the silen ignore @anders-kiaer. Does this look ok? I think the scatter plot matrix can be handled in its own pr, as this one is getting pretty big.

**Comment by anders-kiaer** [comment]
Looks good @kristofferlund. I support scatter plot matrix be in a separate PR :+1:  Is using `print()` to give the warning found to be the best solution, or is [warnings](https://docs.python.org/3/library/warnings.html) something to consider?

**Pull request merged**